content/documentation/rpc/wallet/create_transaction.mdx

```diff
@@ -40,6 +41,8 @@ Creating a raw transaction does not require a spend key. They are not added to t
 }
 ```
 
+> **Note:** outputs.memoHex is a hex encoded buffer with a max length of 32 bytes
```

Comment:
I think this could be a bit confusing since the parameter is a string in the RPC requests

maybe `is a hex-encoded string with a max length of 64 characters`? We could also say that the string is equivalent to a 32 byte buffer

Reply:
Also the memo field has the same limitation, this would make me think you could fit more in the memo field.

Reply:
Fixed